library/CMakeLists.txt

```diff
@@ -173,7 +173,52 @@ if (APPLE OR UNIX)
   endif ()
 endif ()
 
-# testing
+# Testing
 if(BUILD_TESTING)
   add_subdirectory(testing)
 endif()
+
+# Installing
+if(F3D_INSTALL_SDK OR BUILD_SHARED_LIBS)
+  install(TARGETS libf3d
+    EXPORT f3dTargets
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT library
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT library
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT library
+    PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${CMAKE_PROJECT_NAME} COMPONENT headers) #TODO check that
```

Comment:
Todo?

Reply:
useless, removed.

Reply:
turns out it is needed for the SDK, and puttin in SDK specific code is more complex.